lib/bolt/apply_result.rb

```diff
@@ -96,9 +96,9 @@ def initialize(target, error: nil, report: nil)
       @target = target
       @value = {}
       @action = 'apply'
-      value['report'] = report if report
-      value['_error'] = error if error
-      value['_output'] = metrics_message if metrics_message
+      @value['report'] = report if report
```

Comment:
Oh lol so this was...really more of a bug?

Reply:
I guess. :P The implementation class already had the `value` function since it inherits from `Result`. Just needed to expose it on the Puppet side.